content/pages/manual/eyetracking/eyelogic.md

```diff
@@ -0,0 +1,24 @@
+title: EyeLogic
+
+[TOC]
+
+## About EyeLogic
+
+The eye tracking devices from EyeLogic meet the highest standards in accuracy and precision, required for scientific usage. They excel in providing a low and consistent latency, offer strong robustness as well as a high tolerance of head motion. The exceptionality however lies in the combination of quality criteria in connection with portability and ease of use.
+
+- <https://www.eyelogic.de/>
+
+The EyeLogic OpenSesame-plugin is included in the official release of OpenSesame, versions from 3.3.10.
```

Comment:
I assume you mean that EyeLogic is now supported by the PyGaze plugins, right? (Not that there's a separate plug-in.)

Reply:
Yes, I clariefied this sentence (plz see new pull-request).